sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -421,7 +420,9 @@ def _load_from_rest(cls, rest_obj: ComputeResource) -> "ComputeInstance":
             services=[app.as_dict() for app in prop.properties.applications]
             if prop.properties and prop.properties.applications
             else None,
-            created_on=prop.additional_properties.get("createdOn", None),
+            created_on=rest_obj.properties.created_on.strftime("%Y-%m-%dT%H:%M:%S.%f%z")
```

Comment:
Why isn't `created_on` set equal to `rest_obj.properties.created_on` as the else case? Why None as the default?

Reply:
I think the rest object either has created_on as a datetime object that needs to be formatted as a str or None so to me it made sense to implement like this.